source/shared/net/desktop-wallet/remove-baker.rst

```diff
@@ -16,7 +16,7 @@ If you remove a baker in the desktop wallet, the node that is configured with th
 
 #. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Remove baker**.
 
-#. Select the **Identity** that's associated with the baker account whose keys you want to update, and then select **Continue**.
+.. #. Select the **Identity** that's associated with the baker account whose keys you want to update, and then select **Continue**.
```

Comment:
The list numbering is off again.

![image](https://user-images.githubusercontent.com/76045015/123598346-b09cf300-d7f4-11eb-9735-d384a556c914.png)